tests/integrations/python/tests/test_google.py

```
@@ -1678,6 +1678,466 @@ class MathProblem(BaseModel):
+            # First function call should have signature
+            assert len(signatures) > 0, "First function call should have thought signature"
+            assert signatures[0][0] == 0, "Thought signature should be on the first function call part"
+            
+            print("✓ Parallel function call signature handling verified!")
+        else:
+            print(f"  Only {len(function_calls)} function call(s) made")
+            # If only one call, it should still have a signature
+            assert len(signatures) > 0, "Function call should have thought signature"
+
+    @skip_if_no_api_key("gemini")
+    def test_30b_gemini_3_sequential_function_calls_signatures(self, test_config):
```

Ruff (0.14.11)

Unused method argument: `test_config`

(ARG002)